lib/minimax_rb.rb

```diff
@@ -1,5 +1,9 @@
 module MinimaxRB
+  ARGUMENT_MISSING_MESSAGE = "One or more missing arguments!"
+  EXPECTED_ARGUMENTS = [:board, :curr_player_marker, :opponent_marker]
+
   def self.best_move args
```

Comment:
Is there a reason why this is a module with a module method, instead of a class with an instance method?  Both work, but using an instance method would:

* allow you to stop saying `self.` all the time
* allow you to store `empty_marker` as a class variable instead of passing it to each recursion of `best_move`.

Also worth noting: Using a module/class method vs an instance method–in Ruby–doesn't significantly alter the call site, but a static method in a language like C# or Java adds significantly tighter coupling between a call site and a static method because in those languages:

* you can't gracefully (if I remember correctly for C#) specify an interface for static methods -- I don't recall any graceful way to invoke a static method on an abstract class and have it be dispatched to the static method on the concrete class
* you have to hijack the virtual machine / runtime to override a static method from your test, when you try to mock it.  Tools like Powermock use the Proxy pattern and redirect calls that you thought were going to one class/instance, to a whole other proxy.

So it's not a big deal in Ruby since you can say `allow(MinimaxRB).to receive(:best_move) ...`.  But in other languages you'll have a much easier time using an interface that defines an instance method, and passing in a mock instance from your tests.